### PR TITLE
Registrations list performance

### PR DESCRIPTION
**Pull request by jonatanklosko**

### CODE REVIEW

File: WcaOnRails/app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -58,6 +58,8 @@ def psych_sheet_event
   def index
     @competition = competition_from_params
     @registrations = @competition.registrations.accepted.includes(:user, :events).order("users.name")
+    @count_by_event = Hash.new(0)
+    @newcomers = @returners = 0
```
Comment (jfly) [inline WcaOnRails/app/controllers/registrations_controller.rb]: IMO, I don't think there's any value in putting these here as opposed to local variables inside of the view.

Comment (jonatanklosko) [inline WcaOnRails/app/controllers/registrations_controller.rb]: I just feel like there's not reason for adding even more logic to the view, whereas the controller action is almost empty =)

Comment (jfly) [inline WcaOnRails/app/controllers/registrations_controller.rb]: Ok, if you like it this way, let's do it this way!

**Comment by larspetrus** [comment]
It's a little awkward that we're changing the UI at the same time as speeding things up. I like both changes, especially removing the useless and space hogging WCA ID, so it seems OK to me, but I just wanted to mention it.

LGTM when you feel ready.

**Pull request merged**